docs/Backups.md

```diff
@@ -183,6 +183,10 @@ Homestead, Homer or Memento will produce output of the following form.
     sip_digests: Deleting old .db files...
     sip_digests: Restoring from backup: 1372336442947
 
+For Homestead, Homer or Memento, after restoring a backup you must also repair the node. To do this:
```

Comment:
Saying 'you must repair the node' suggests that it is currently broken, which i don't think we want to imply. Would be better if it explicitly stated you're running the cassandra command nodetool repair, to verify that the backup is healthy (or whatever it is that repair is actually doing)